app/services/cfe_param_builders/regular_transactions.rb

```diff
@@ -48,8 +48,8 @@ def self.build_payments(cfe_translations, form, operation)
     end
 
     def self.value(form, local_name)
-      if FeatureFlags.enabled?(:conditional_reveals, form.check.session_data) && form.is_a?(OtherIncomeForm)
-        form.send("#{local_name}_conditional_value") if form.send("#{local_name}_received")
+      if FeatureFlags.enabled?(:conditional_reveals, form.check.session_data)
```

Comment:
question - Should we use a ternary operator to send `0`, as previously user's had to enter `0` for all the values? I guess it's not breaking at the moment when sending to CFE.

Reply:
This class is already filtering out zeroes (see above `.select { |_cfe_name, local_name| value(form, local_name).to_i.positive? }`, so it makes no ends if we return nil or 0 here - it'll be filtered out regardless.

Reply:
Oh yeah, that makes sense. I didn't scroll up.